dot/parachain/availability-store/availabilitystore.go

```diff
@@ -57,33 +183,93 @@ func (av *AvailabilityStoreSubsystem) processMessages() {
 }
 
 func (av *AvailabilityStoreSubsystem) handleQueryAvailableData(msg QueryAvailableData) {
-	// TODO: handle query available data
+	result, err := av.availabilityStore.loadAvailableData(msg.CandidateHash)
+	if err != nil {
+		logger.Errorf("failed to load available data: %w", err)
+	}
+	msg.Sender <- result
 }
 
 func (av *AvailabilityStoreSubsystem) handleQueryDataAvailability(msg QueryDataAvailability) {
-	// TODO: handle query data availability
+	_, err := av.availabilityStore.loadMetaData(msg.CandidateHash)
+	if err != nil {
+		msg.Sender <- false
+	} else {
+		msg.Sender <- true
+	}
```

Comment:
```suggestion
msg.Sender <- err == nil
```

Reply:
We should also return this error.
We could ignore `pebble: not found` error, but should forward any other errors. And then log these errors in processMessages